src/isar/state_machine/transitions/functions/stop.py

```diff
@@ -31,7 +31,9 @@ def stop_mission_cleanup(state_machine: "StateMachine") -> bool:
     stopped_mission_response: ControlMissionResponse = (
         state_machine._make_control_mission_response()
     )
-    state_machine.events.api_requests.stop_mission.output.put(stopped_mission_response)
+    state_machine.events.api_requests.stop_mission.response.put(
+        stopped_mission_response
+    )
```

Comment:
Good example why this renaming is a good idea; the input variable was already named response :)